islandora_bookmark.module

```diff
@@ -197,6 +201,29 @@ function islandora_bookmark_block_view($delta = '') {
         $block['pid'] = $pid;
       }
       break;
+
+    case 'islandora_bookmark_list_info':
+      // Block intended for use on the list view page.
+      // EX: 'islandora-bookmark/listid/1'.
+      if (arg(0) == 'islandora-bookmark') {
+        $uri = url(current_path(), array('absolute' => TRUE));
+        $list_id = arg(2);
+        if ($list_id) {
+          $bookmark = Bookmark::getList($list_id);
+          $user = user_load($bookmark->bookmarkOwner);
+          $block['content'] = theme(
+            'islandora_bookmark_list_info',
+            array(
+              'name' => $bookmark->bookmarkName,
```

Comment:
Passing the bookmark list object and calculating things in either an implementation of either template_preprocess or template_process seems like it would be more useful... Also, the ability to get at the object easily, if one really had to.